tests/e2e/helpers.js

```diff
@@ -37,6 +37,31 @@ async function waitFor(check, iterations = 10, timeout = 1000) {
   console.error("Condition was not meet in time")
   process.exit(2) // exiting here so the e2e tests actually fail. if not they pass
 }
+
+async function fundMasterAccount(browser, network, address) {

```

Comment:
how does this scale?? will you create a funding function for every network? why not stick with the wallet strategy from before?

Reply:
Yes, I suppose to or it could just notify us somehow. This function only trigger when master account has a low balance. Newly created accounts are still funded from master, this function is only for funding master account

Reply:
this only works for gaia testnet is what I meant. so this should if we want to keep this be refactored into some structure that tells us this only handles gaia testnet.